test/test-local.sh

```diff
@@ -8,6 +8,19 @@
 #     .\_/.
 #
 
+INITIAL_IMAGE_COUNT=$(docker images | grep -c aegis)
+
+if [ "$INITIAL_IMAGE_COUNT" -eq 0 ]; then
+    echo ""
+    echo "There are no Aegis images in the registry."
+    echo "Are you sure you are using the minikube docker?"
+    echo ""
+    echo "Also make sure you have executed 'eval \"\$(minikube docker-env)\'"
+    echo "before building images."
+    echo ""
+    exit 1
```

Comment:
I pushed to the wrong registry more than once.

So I’m putting guards in scripts to prevent that when possible.